.semaphore/semaphore.yml

```diff
@@ -52,12 +52,12 @@ blocks:
       - export BRANCH_NAME=$SEMAPHORE_GIT_BRANCH
       - if [ -z "${SEMAPHORE_GIT_PR_NUMBER}" ]; then make image-all cd CONFIRM=true; fi
 
-# build windows release
-- name: "nightly build windows release"
+# build windows archive
```

Comment:
We could probably remove this comment

Reply:
I will merge it as it is since it takes long time to go through CI. Will remove that comment on my next PR for aws routes.